service/haproxy/run

```diff
@@ -33,7 +33,7 @@ reload() {
 
     # Trigger reload
     LATEST_HAPROXY_PID=$(cat $PIDFILE)
-    haproxy -p $PIDFILE -f /marathon-lb/haproxy.cfg -D -sf $LATEST_HAPROXY_PID 200>&-
+    /marathon-lb/haproxy_wrapper.py `which haproxy` -p $PIDFILE -f /marathon-lb/haproxy.cfg -D -sf $LATEST_HAPROXY_PID 200>&-
```

Comment:
do not fork this, must block the execution until reload is over.
test the return code, should log to stderr in case of error.